### ISSUE TITLE
Map Location autocomplete dropdown

### ISSUE THREAD

**Issue by toleabivol** [ThemeFuse/Unyson#2598]
Hello,
the autocomplete dropdown location field in the map shortcode is not working anymore ( the dropdown does not appear ).
No errors in the console and gapi key inserted.

**Comment by GheorgheP** [comment]
Please create a content backup of the installation in provide it so I would test on my local machine.
You can submit it privately to me in `pisarenco[[dot]]gh[[at]]gmail[[dot]]com`

**Comment by arvi** [comment]
@GheorgheP I am experiencing this too. Wordpress 4.7.5 and Unyson Version 2.6.16. I typed in my address at home, it cannot find it. I also try to use https://fakena.me/random-real-address/ but marker is just stuck at the sea. Can you give me an address that I can test if working, which is working on your end?

**Comment by toleabivol** [comment]
@GheorgheP did you get my email ? Sent 3 days ago

**Comment by arvi** [comment]
@toleabivol Finally resolved my issue by just getting a new api key and replacing my old one... maybe try if it also work on your end. It is now autocompleting. https://developers.google.com/maps/documentation/javascript/get-api-key

**Comment by GheorgheP** [comment]
@toleabivol I did not get it. But try to change the API key like @arvi did. Our guys from technical support confirmed that they receive this issue from our clients too, and it's necessary to change the AI key.
This may be because you reach the API request limits.

**Comment by toleabivol** [comment]
Somehow it started working again with the same old API key so I will just close it.

**Comment by arvi** [comment]
Note to self:

I had this issue once again (Unyson Version 2.6.16), and for some reason it cannot pick up the key via `FW_Option_Type_GMap_Key::get_key()` from the api key field on the map shortcode. I am getting `MissingKeyMapError` and `NoApiKeys` and `RetiredVersion`. 

I decided to just hard code the key on the ff. to keep error from persisting from time to time:

`wp-content/plugins/unyson/framework/includes/option-types/map/class-fw-option-type-map`

and 

`themes/theme-name/framework-customizations/extensions/shortcodes/shortcodes/map/static.php`

I also changed the version from 3.23 to 3.30.
